modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -94,6 +93,54 @@ def create
 
         private
 
+        def validate_decide_params!(proc_id:, decision:)
+          if proc_id.blank?
+            raise ::Common::Exceptions::ParameterMissing.new('procId',
+                                                             detail: 'procId is required')
+          end
+
+          unless decision.present? && %w[accepted declined].include?(decision)
+            raise ::Common::Exceptions::ParameterMissing.new(
+              'decision',
+              detail: 'decision is required and must be either "ACCEPTED" or "DECLINED"'
+            )
+          end
+        end
+
+        def send_declined_notification(ptcpnt_id:, first_name:, representative_id:)
+          lockbox = Lockbox.new(key: Settings.lockbox.master_key)
+          encrypted_ptcpnt_id = lockbox.encrypt(ptcpnt_id)
+          encrypted_first_name = lockbox.encrypt(first_name)
+
+          ClaimsApi::VANotifyDeclinedJob.perform_async(encrypted_ptcpnt_id:, encrypted_first_name:,
+                                                       representative_id:)
+        end
+
+        def validate_ptcpnt_id!(ptcpnt_id:, proc_id:, representative_id:, service:)
+          if ptcpnt_id.blank?
+            raise ::Common::Exceptions::ParameterMissing.new('ptcpntId',
+                                                             detail: 'ptcpntId is required if decision is declined')
+          end
+
+          if representative_id.blank?
+            raise ::Common::Exceptions::ParameterMissing
+              .new('representativeId', detail: 'representativeId is required if decision is declined')
+          end
+
+          res = service.read_poa_request_by_ptcpnt_id(ptcpnt_id:)
+
+          raise ::Common::Exceptions::Lighthouse::BadGateway if res.blank?
+
+          poa_requests = Array.wrap(res['poaRequestRespondReturnVOList'])
+
+          matching_request = poa_requests.find { |poa_request| poa_request['procId'] == proc_id }
```

Comment:
When I was trying to test this locally I noticed that it looks like the value comes back as `procID` ?  

<img width="539" alt="Screenshot 2024-11-15 at 9 24 28 AM" src="https://github.com/user-attachments/assets/9cf10297-e5bb-4475-8bec-1ffab047e931">

Reply:
I was also a little confused, and this could just be my local testing I am not super familiar with this workflow so trying to learn it here, but I thought I should have had a match in this case?  Am O misunderstanding something?

<img width="773" alt="Screenshot 2024-11-15 at 9 29 30 AM" src="https://github.com/user-attachments/assets/523225c8-6a5b-418f-b239-c4fb1fe79a53">

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/19362/commits/1800d14ffa462b1adbbb703259e9193a108b90f5